lib/view_component/test_helpers.rb

```diff
@@ -103,11 +117,12 @@ def render_preview(name, from: __vc_test_helpers_preview_class, params: {})
     #
     # assert_text("Hello, World!")
     # ```
-    def render_in_view_context(*args, &block)
+    def render_in_view_context(*, &block)
       @page = nil
-      @rendered_content = vc_test_controller.view_context.instance_exec(*args, &block)
+      @rendered_content = vc_test_controller.view_context.instance_exec(*, &block)
```

Comment:
Why did this change?

Reply:
pretty certain Standard did that, in any case I can revert (may have been a config on my end).